### DISCUSSION TITLE
Context menu for titles

### DISCUSSION THREAD

**Discussion by PavanLuca** [bleachbit/bleachbit#1652] [Ideas]
Add context menu even to titles.

**Comment by az0** [comment]
Could you please elaborate. Which titles? An annotated screenshot may help.

What do you want to see in the the context menu?

**Reply by PavanLuca** [reply]
Sorry, maybe I should have said drop-down buttons 😅.
![Schermata del 2025-03-15 22-41-42](https://github.com/user-attachments/assets/bff0f696-e167-43e7-a923-0e031d67db18)
If they had the context menu users can scan or delete a certain section rather than choosing the entries one by one. Of course only the marked entries will be scanned or deleted. This small enhancement is to let the user's favorite selection as is but clean only some specific sections. For example a user usually lets all cleaners selected but in some cases wants to clear only a certain section. To do so it has to scan or clean every entry of that section manually, one by one, but setting the context menu even to the drop-down buttons will allow a faster action.

**Reply by az0** [reply]
Oh, I understand. Right now the context menu exists for the options like Cache, but you want it at the higher level like Firefox. That makes sense.

Remind me in a future release when you see more work on the [next-generation GUI](https://github.com/bleachbit/bleachbit_gui_next_gen).  I don't want to add it yet to the current GUI, which will be replaced.

**Reply by PavanLuca** [reply]
Will you also add an option to enable the current GUI when you release the new one? I think that letting users choose is better, both for preference and habit.

**Reply by az0** [reply]
I've been considering giving users the option. As I work on integrating it, I'll know more of how well it works to ship both. 

One idea was to ship the new one as a demo even before it works at all, like shown in the video I posted of the new GUI
